packages/design-system-tokens/src/themes/healthcare-components.ts

```diff
@@ -57,7 +57,7 @@ export const components: AnyTokenValues = {
 
   'button': {
     '__background-color--active':                 t.color['primary-darkest'],
-    '__background-color--disabled':               'transparent',
+    '__background-color--disabled':               t.color['white'],
```

Comment:
@line47 Requested updated to disabled background. Change from `transparent` to `white` for all contexts (light/dark).